src/visualization/client/Turi Create Visualization/src/user_interface/src/elements/Annotate/SingleImage/SimilarImage/index.js

```diff
@@ -21,7 +18,8 @@ class SimilarImage extends Component {
              onClick={this.props.onClick.bind(this, this.props.src.index)}>
           <img src={this.props.src.src}
                className={style.SingleImageIndex}
-               style={this.resizeImage(this.props.src.width, this.props.src.height)} />
+               style={this.resizeImage(this.props.src.width, this.props.src.height)} 
+	       alt="Can't Find" />
```

Comment:
Let's leave the alt text blank (empty string), since there isn't a good alternate explanation for the contents of the image -- I don't think "Can't Find" is quite correct, since for a user with a screen reader, the image is found, it's just that they can't see it.